### PR TITLE
Create .ubeu TLD

### PR DESCRIPTION
**Pull request by BGC-ubeu**

### CODE REVIEW

**Comment by kenthejr** [comment]
Just for my own understanding, did you intend for the net revenue distribution to go:
- 50% DEV
- 0% COOP
- 50% TLD ADMIN

I know previous KAPs used that model, but I also know that .hh and .ℏ have it set to 50/25/25.

This is completely up to you (@BGC-ubeu) and the community to decide, I just want to make sure it was intentional before we launch it.

**Comment by BGC-ubeu** [comment]
Thanks for the feedback. The intention was to follow the standard revenue distribution (50/25/25), and the KAP has been updated to reflect that. Thanks again.

**Review by kenthejr — APPROVED** [review]

File: .ubeu TLD [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Specification
+A top-level domain (TLD) '.ubeu' smart contract maintains the registry of domains, records the owner, the resolver, and allows the owner to make changes to the data therein. A registrar smart contract is written owning the secondary-level domains and issues subdomain NFTs of '.ubeu' domains to buyer. Domain resolution is made using KNS resolver. The subdomain NFT allows its owner to interact with the UBeU app and create secondary subdomains.
+
+The UBEU TLD should be issued as `.ubeu`.
+
+This TLD will be issued as a KNS TLD NFT.
+
+The admin of this TLD shall be 0.0.3724158.
+
```
Comment (BGC-ubeu) [inline .ubeu TLD:43]: The '.ubeu' TLD will be administered as a private TLD, giving the admin sole authority to register domains under the '.ubeu' TLD.

**Review by kenthejr — APPROVED** [review]

**Comment by tonycamero** [comment]
How does this support 3rd party DID/SSID providers?

**Comment by BGC-ubeu** [comment]
> How does this support 3rd party DID/SSID providers?

I don't see any immediate connection between the UBeU proposal and DID/SSID identity forms - if there is something specific you have in mind let me know. The UBeU model is leveraged to capitalize on and provide a solution to the confluent trends of proliferating Web3 digital identity needs and the rapid tokenization of real-world assets. And, by virtue of network economics, I have a strong conviction that Hedera will be a primary foundation for these emerging trends. 

I also think that digital identity tokens in standardized formats utilizing 'first/username.lastname' formats - while an antithesis of early DLT adopters who prioritized non-conformity and anonymity - is going to become inevitable as more casual users are drawn to Web3 applications, and due to the utility that digital ID standardization can provide. I describe the format as '*.first/username.lastname.ubeu' but another way to consider it is 'resource.labelA.labelB.ubeu.'  The "resource" can be a specific DApp which, by interacting with the UBeU registry, can create these tokens mapped to its resources, or the "resource" can be a custom name defined at the UBeU app by the digital ID holder (i.e. TurtleMoon, defi, wallet, or "Assets4Sale") which can then be used to partition assets among wallets. From a developer perspective, I think this standardization also allows for more user-friendly and personalized apps. Sorry for the tangent.

Again, let me know if there is something particular with respect to 3rd party DID/SSID that caused me to miss the mark with my answer and I am happy to follow-up.

**Review by tonycamero — APPROVED** [review]

File: .ubeu TLD [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+kap: 
+title: .ubeu TLD
+type: Information
+subproject: KNS
+sponsor: Hedera Domains Service - UBeU.io
+author: Brian G Coleman
```
Comment (kenthejr) [inline .ubeu TLD:7]:
```suggestion
author: Brian G Coleman <@BGC-ubeu>
```

File: .ubeu TLD [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+status: Draft
```
Comment (kenthejr) [inline .ubeu TLD:8]:
```suggestion
status: Final
```

File: .ubeu TLD [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+created: 2023-09-24
+updated: 2023-10-09
```
Comment (kenthejr) [inline .ubeu TLD:10]:
```suggestion
updated: 2023-10-16
```

File: .ubeu TLD [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+deadline:
```
Comment (kenthejr) [inline .ubeu TLD:11]:
```suggestion
```

**Review by kenthejr — APPROVED** [review]
Update for final commit.

**Pull request merged**